components/button.css

```diff
@@ -48,22 +50,38 @@
   background-color: var(--bg-dark);
   color: var(--txt-light);
 }
-.btn-outline-primary {
+.btn-outline {
   background-color: transparent;
   border-color: #ff584f;
   color: var(--txt-dark);
 }
-.btn-outline-primary:hover {
+.btn-outline:hover {
   background-color: #ff584f;
   transition: all 0.5s;
   color: var(--txt-light);
 }
 .btn-floating {
+  position: fixed;
+  z-index: 10;
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
   border-radius: var(--border-circle);
   padding: 0.5rem;
   width: 3rem;
 }
 .btn-floating-danger {
+  position: fixed;
+  z-index: 10;
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
   border-radius: var(--border-circle);
   padding: 0.5rem;
   width: 3rem;
```

Comment:
Both the classes have same properties  you can merge them code is getting repeated

Reply:
Done